src/components/guide/guide.js

```diff
@@ -577,7 +577,7 @@ define(['require', 'inputManager', 'browser', 'globalize', 'connectionManager',
                     html += '</div>';
 
                     if (program.IsHD && options.showHdIcon) {
-                        //html += '<i class="guideHdIcon material-icons programIcon">hd</i>';
+                        //html += '<span class="guideHdIcon material-icons programIcon hd"></span>';
```

Comment:
Not strictly in the scope of this PR, but any clue why we aren't using this icon? I think it would be really useful to check which channels are HD at a glance, no clue why it's not used honestly.

Not an user of Live TV, so you might know better.

Reply:
First appearance de6ac33ec186d63eaf2e77800a61d2d743357edd
This is probably supposed to be an alternative to text markers. cc @JustAMan

Reply:
Maybe we can try uncommenting it here

Reply:
![hd_icon](https://user-images.githubusercontent.com/56478732/80790585-ef56ff00-8b97-11ea-9d9b-495beaf2d0b0.png)
left - icon, right - text